2016-10-16-why-mocking-fails.md

```diff
@@ -0,0 +1,50 @@
+# Why mocking fails?
+
+[Yegor arguments that fake objects are better than mocks](http://www.yegor256.com/2014/09/23/built-in-fake-objects.html) shows that mocks can be really hard to build when objects create another objects and these another objects create more objects and so on (maybe forever?) and I agree with it (this is the beginning of a **mock hell**).
```

Comment:
The plural makes it tricky, but I think the correct form would be:

"_**Yegor's** arguments..._"